st2common/st2common/models/api/execution.py

```diff
@@ -121,6 +137,9 @@ def from_model(cls, model, mask_secrets=False):
             end_timestamp = isotime.format(end_timestamp, offset=False)
             doc['end_timestamp'] = end_timestamp
 
+        for entry in doc['log']:
```

Comment:
It seems like mongoengine does it for us since we've defined the field to be `me.ListField(field=me.DictField())`. I'm going to play safe and update it per your request anyway.